src/SymbolDirectories/Manager.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "SymbolDirectories/Manager.h"
+
+#include <QSettings>
+
+constexpr const char* kSymbolDirectoriesSettingsKey = "symbol_directories";
+constexpr const char* kDirectoryPathKey = "directory_path";
```

Comment:
nit: constexpr std::string_view.

Reply:
I prefer to not use std::string_view here. This is used for `QSettings::beginReadArray(const QString&)`. And QString cannot handle std::string_view.